src/linearmixedmodel.jl

```diff
@@ -169,38 +161,16 @@ function LinearMixedModel(
     )
 end
 
-function fit(
-    ::Type{LinearMixedModel},
-    f::FormulaTerm,
-    tbl;
-    wts=[],
-    contrasts=Dict{Symbol,Any}(),
-    progress::Bool=true,
-    REML::Bool=false,
-)
-    return fit(
-        LinearMixedModel,
-        f,
-        Tables.columntable(tbl);
-        wts=wts,
-        contrasts=contrasts,
-        progress=progress,
-        REML=REML,
-    )
+function fit(::Type{LinearMixedModel}, f::FormulaTerm, tbl;
+             wts=[], contrasts=Dict{Symbol,Any}(), progress::Bool=true,
+             REML::Bool=false, σ=nothing)
+    return fit(LinearMixedModel, f, Tables.columntable(tbl);
+               wts, contrasts, progress, REML, σ)
 end
```

Comment:
These formatting changes might be undone by `format` with BlueStyle

Reply:
This might be a good time to go ahead and add a JuliaFormatter CI step -- I'll look into it.